src/main/java/core/basesyntax/service/operation/impl/OperationReturnHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.operation.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+
+public class OperationReturnHandler implements OperationHandler {
```

Comment:
Consider renaming the class to `ReturnOperationHandler` for better naming consistency.